scripts/start-qemu-daemon

```diff
@@ -0,0 +1,122 @@
+#!/bin/sh
+
+# usage:
+#  start-qemu [<image-name>]
+#
+# if image name is not provided, the script will attempt to run
+# genivi-dev-platform
+#
+# QEMU will be started in daemonized mode and the PID will be written to
+# /tmp/qemu.pid
+#
+# Additional ENV variables can be set to control start-up:
+#
+#    MACHINE                - Yocto target MACHINE
+#    QEMU_DRIVE             - "-drive" argument to qemu
+#    QEMU_GRAPHICS_ENABLE   - enable SDL graphics with HW acceleration
+#    QEMU_AUDIO_ENABLE      - Will enable audio in qemu (-soundhw ac97)
+#    DISK_IMG               - .ext4 image to use in "-drive" argument
+#    KERNEL_IMG             - path to Linux kernel (-kernel)
+#
+# NOTE: current settings
+#           network: TAP 192.168.7.2::192.168.7.1
+#           VNC: port 23
+#           serial console: stdio
+
+set -e
+set -x
+
+# if called under Yocto/OE shell, BUILDDIR will already be set correctly
+if [ -z "$BUILDDIR" ]; then
+    # try to guess BUILDDIR
+    BUILDDIR="`dirname $0`"/../gdp-src-build
+fi
+
+IMAGE_NAME=$1
+# check if it's named like an image
```

Comment:
You're not checking this so I'd remove the comment.

Reply:
Will clean-up